### ISSUE TITLE
Chemical notations that need to be accepted

### ISSUE THREAD

**Issue by NSoiffer** [daisy/MathCAT#73]
From IUPAC's Red Books (2005):

Formulae of optically active compounds
> The sign of optical rotation is placed in parentheses, the wavelength (in nm) being indicated
> as a right subscript. The whole symbol is placed before the formula and refers to the sodium
> D-line unless otherwise stated.
> Example:  $(+)_{589}-[\rm{Co(en)}_3]Cl_3$

Indication of excited states
>Excited electronic states may be indicated by an asterisk as right superscript. This practice
> does not differentiate between different excited states.
> Example $\rm{He}^*$

Structural descriptors
> Structural descriptors such as cis, trans, etc., are listed in Table V. Usually such descriptors
> are used as italicized prefixes and are connected to the formula by a hyphen.
> Examples:
> 1. cis-[PtCl2(NH3)2]
> 2. trans-[PtCl4(NH3)2]

**Comment by NSoiffer** [comment]
Neil needs to add yet another Chemistry notation for precipation(?)

**Comment by rileeki** [comment]
According to Neil, this may be difficult for someone else to pick up as a small, isolated task because the chemistry code currently relies on a number of heuristics and is somewhat hard to follow.

That said, he thought it could be valuable for someone who is interested in spending time learning that part of the codebase. Someone who took the time to understand, document, and clean it up could make a meaningful contribution.

**Comment by NSoiffer** [comment]
From the French braille code spec, here's another (2) notation that should be recognized:

<img width="626" height="591" alt="Image" src="https://github.com/user-attachments/assets/d310faa4-b254-4ba2-87d7-a8259b3d3348" />

The downarrow is ↴ (U+21B4). The uparrow is ⬏ (U+2B0F).
